client/components/split-button/docs/example.jsx

```diff
@@ -32,15 +32,16 @@ class SplitButtonExample extends PureComponent {
 		const compact = { compact: this.state.compactButtons };
 		return (
 			<div>
-				<a className="docs__design-toggle button" onClick={ this.toggleButtons }>
+				<button className="docs__design-toggle button" onClick={ this.toggleButtons }>
 					{ this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons' }
-				</a>
+				</button>
 				<Card>
 					{ map(
 						[
 							[
 								{ label: 'Split Button', icon: 'history' },
 								{ label: 'Split Button', primary: true },
+								{ label: 'Split Button with whtie separator', primary: true, separatorWhite: true },
```

Comment:
We need to fix the typo here. Also, we should fix the separator prop name as it does not match implementation - probably a refactoring issue:

```suggestion
								{ label: 'Split Button with white separator', primary: true, whiteSeparator: true },
```

Reply:
Thank you!, good catch!! 💪